docs/translations/translations.json

```diff
@@ -66,7 +66,7 @@
   "copySourceLinkJS": "Copy link to JavaScript source",
   "copySourceLinkTS": "Copy link to TypeScript source",
   "stackblitz": "Edit in StackBlitz (JS only)",
-  "headTitle": "The world's second most popular React UI framework - Material-UI",
+  "headTitle": "Material-UI: A React UI component framework",
```

Comment:
It doesn’t really stand out 🤔

Reply:
"Production ready react UI components"?

Reply:
Alright, I think that we can go for

"Material-UI: Ready-made React UI component framework"

The idea comes from @joshwooding, @eps1lon was supportive for it, I think that ready-made is a contraction of "faster and easier web development." that @mbrookes first proposed.